tests/ops/util/test_tcf_util.py

```diff
@@ -0,0 +1,578 @@
+import pytest
+from fideslang import MAPPED_PURPOSES
+from fideslang.models import LegalBasisForProcessingEnum
+
+from fides.api.models.sql_models import PrivacyDeclaration
+from fides.api.schemas.tcf import EmbeddedVendor
+from fides.api.util.tcf_util import get_tcf_contents
+
+
+class TestTCFContents:
+    def test_load_tcf_data_uses_no_applicable_systems(self, db):
+        get_tcf_contents.cache_clear()
```

Comment:
nit: could we make this operation an auto-use class fixture so that it's automatically run before every test in the class? just to make this a bit more robust going forward! i think that should be simple, if it's not then just ignore me.

Reply:
Sure I can look into that.  What do you think about the `lru_cache` I'm using here to begin with though? Any concerns?

Reply:
oh thanks for bringing my attention back to that! 👍 

on the face of it, it feels like a good idea, to repeating those many operations involved in computing the overlay. but on closer look i'm questioning whether the cache would be invalidated appropriately? my understanding of the LRU cache is that its caching based on the hash of the function arguments. in this case, the lone argument is a DB `Session` - would its hash be updated when the relevant DB state (in this case, datamap contents) changes? because i do think we'd want to rerun the function on basically any update/addition/removal of a `System` in our datamap, right?

maybe i'm thinking about this wrong! don't have a ton of experience with LRU caches in python. i suppose one thing that would settle my concerns indefinitely, and would be nice to have moving forward, is an explicit test that the cache is _not_ getting in the way of providing up-to-date, accurate results. what do you think?

Reply:
I've been mulling this over,:I'm thinking I should just remove the caching altogether.  A `session` argument is not a great one to determine if the cache should be invalidated.  It's hard to unit test because we're literally passing around the same session within the test. The session is the same, so the cache isn't invalidated, hence the `cache_clear()`.

However, testing locally against a running instance, subsequent requests seem to recalculate system updates just fine, and I think it has nothing to do with the system changing, and something about the session object changing. I imagine it's recalculating it every time, so having little practical effect.

Reply:
yeah that makes a lot of sense to me, feels like at this point it'd only add confusion, and we can target any optimizations as needed in a future iteration, via an LRU cache or other mechanism 👍

Reply:
lru cache removed! This was something I'd added very early on and hadn't thought about it since, so I'm glad we revisited before merge. thanks for the comments on this test that reminded me!